pkgs/os-specific/linux/net-tools/default.nix

```diff
@@ -15,6 +15,8 @@ stdenv.mkDerivation rec {
     '';
 
   makeFlags = [
+    "CC=${stdenv.cc.targetPrefix}cc"
+    "AR=${stdenv.cc.targetPrefix}ar"
```

Comment:
These don't appear to be needed either. Same command as above, but try building nettools.

Reply:
Hmm, I could have sworn I cherry-picked this one for a reason but indeed I haven't seen a failure yet after reverting it. Very odd.